src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -137,6 +144,26 @@ def _add_input_args(parser):
         help=f"The number of unique prompts to generate as stimulus.",
     )
 
+    input_group.add_argument(
+        "--output-tokens-mean",
+        type=int,
+        default=LlmInputs.DEFAULT_OUTPUT_TOKENS_MEAN,
+        required=False,
+        help=f"The mean number of tokens in each output. Ensure the --tokenizer value is set correctly. "
+        "Note that there is still some variability in the requested number of output tokens, but GenAi-Perf "
+        "attempts its best effort with your backend, endpoint, and model to get the right number of output tokens. "
+        "For the TRT-LLM backend, you currently must set 'exclude_input_in_output' to true in the model config to "
+        "not echo the input tokens in the output.",
+    )
+
+    input_group.add_argument(
+        "--output-tokens-stddev",
+        type=int,
+        default=LlmInputs.DEFAULT_OUTPUT_TOKENS_STDDEV,
+        required=False,
+        help=f"The standard deviation of the number of tokens in each output. This is only used when output-tokens-mean is provided.",
```

Comment:
can we restrict the description of the new args to 80 characters?

Reply:
Per line or per description? I missed the per-line, because I thought pre-commit would catch it. Maybe it ignores strings. Will push the change in a second. If you want me to shorten the description under 80 characters, that'll be harder, but let me know.

Reply:
I meant per line 😝 sorry for confusion

Reply:
No worries, thanks for clarifying!